src/app/Routing.tsx

```diff
@@ -22,6 +23,19 @@ const Routing: React.FC = () => {
       <Route path="/register" element={<LoginPage />} />
       <Route path="/register/form" element={<LoginFormPage />} />
       <Route path="/404" element={<NoMatch />} />
+      <Route
+        path="/profil"
+        element={
+          <ProfilPage
+            accountType={"student"}
+            name={"Jan"}
+            surname={"Kowalski"}
+            birthDate={"2137"}
+            email={"q@2wp.pl"}
+          />
+        }
+      />
+      {/* to przekazywanie danych będzie trzeba jakoś zmienić pewnie */}
```

Comment:
about this, it's fine for now there's high probability that we'll fetch the data in loader and pass it then by props. Overally gj, it's dumb component (as it should be because it can)